setup/test_utils.py

```
@@ -0,0 +1,217 @@
+# Add project root to path for imports
+sys.path.insert(0, str(Path(__file__).resolve().parent.parent))
+
+from setup import utils
+
+
+class TestFindProjectRoot:
+    """Tests for find_project_root function."""
+
+    def test_find_project_root_with_marker(self):
+        """Test finding project root with existing marker."""
+        with patch.object(Path, 'exists', return_value=True) as mock_exists:
```

Flake8 (7.3.0)

[error] local variable 'mock_exists' is assigned to but never used

(F841)